collectors/python.d.plugin/zoneminder/zoneminder.chart.py

```diff
@@ -0,0 +1,188 @@
+                if ("<error>" in result):
+                    self.debug("error: " + output)
+                    return None
+                access_token = output
+        
+        #get data from monitors api call
+        try:
+            r = requests.get(self.zoneminder_url + '/api/monitors.json?token=' + access_token, timeout=self.connection_timeout, verify=False)
+            json_data = r.json()
+        except requests.exceptions.RequestException as e: 
+            self.debug(e)
+            return None
+
+        if all (k in json_data for k in ("success","data")):
+            if (json_data['success'] == False and 'Token revoked' in json_data['data']['name']):
+                self.debug("token was revoked, generating new tokens, will try to collect data in next run...")
+                result,output = zm_generate_refresh_token(self.zoneminder_url, self.zm_user, self.zm_password, self.connection_timeout)
+                if ("<error>" in result):
+                    self.debug("error: " + output)
+                return None
+               
+        if ("monitors" in json_data):
+            for monitor in json_data["monitors"]:
+                if ("Monitor" in monitor):
+                    try:
+                        disk_space += float(monitor["Monitor"]["TotalEventDiskSpace"])
+                    except Exception as e:
+                        self.debug(e)
+                    if (monitor["Monitor"]["Function"] == "None" or monitor["Monitor"]["Enabled"] == "0"):
+                        continue
+                    if "zm_fps_" + monitor["Monitor"]["Id"] not in self.charts['camera_fps']:
+                        self.charts['camera_fps'].add_dimension(["zm_fps_" + monitor["Monitor"]["Id"], monitor["Monitor"]["Name"], 'absolute'])
+                    if "zm_bandwidth_" + monitor["Monitor"]["Id"] not in self.charts['camera_bandwidth']:
+                        self.charts['camera_bandwidth'].add_dimension(["zm_bandwidth_" + monitor["Monitor"]["Id"], monitor["Monitor"]["Name"], 'absolute', None, 1024])
+                    if "zm_events_" + monitor["Monitor"]["Id"] not in self.charts['events']:
+                        self.charts['events'].add_dimension(["zm_events_" + monitor["Monitor"]["Id"], monitor["Monitor"]["Name"], 'absolute'])
+                    try:
+                        data["zm_fps_" + monitor["Monitor"]["Id"]] = float(monitor["Monitor_Status"]["CaptureFPS"])
+                        data["zm_bandwidth_" + monitor["Monitor"]["Id"]] = float(monitor["Monitor_Status"]["CaptureBandwidth"])
+                        data["zm_events_" + monitor["Monitor"]["Id"]] = float(monitor["Monitor"]["TotalEvents"])
+                    except Exception as e:
+                        self.debug(e)
+        else:
+            self.debug("Invalid zoneminder api response: " + r.text)
+            return None
+
+        data["zm_disk_space"] = disk_space
+
+        return data
```

Comment:
Github is saying that we do not have a new line at end of file, please, add it one line here.